### ISSUE TITLE
"Edit credit card/PayPal" is awkwardly-worded

### ISSUE THREAD

**Issue by tofumatt** [mozilla/payments#115]
In the [payments workflow](https://raw.githubusercontent.com/mozilla/payments/brampitoyo-patch-8/docs/design/ux-pay-wireframe.png), we prompt the user to "_Edit credit card/PayPal_" (or "Add credit card/PayPal" if they only have one payment method), but I think this is a bit awkwardly-worded. It's also longer than "**Edit payment methods**" (or even "✎ Payment methods" or something).

Unless we want to explicitly call out the methods we support (credit card or PayPal), I think "Payment method" is a better noun, so we could **change this string to "Edit payment methods"**. It's also future-proof; if we add DogeCoin or whatever in the future we don't need to amend the string, and it may confuse users with Visa Debit cards less, as I don't think of that as a credit card.

**Comment by brampitoyo** [comment]
The reason why I want to stick to “Add credit card/PayPal” (or Edit, or Delete, and so on) is to make the interface as familiar as possible to the user. If we only support two methods, why not spell them out? To me, it’s not a category unless we have at least 3 members in the set.

But you do have a point about conflating debit and credit cards. Amazon, for example, worded it as “Add a Credit or Debit Card”. If we follow their example, our heading becomes “Add a Credit or Debit Card, or PayPal”, and that’s really confusing!

So perhaps we could change this string to “Add Credit/Debit Card or Paypal”? It’s a bit wordy, but it works for the time being. When we support BitCoin, we should definitely change it to “Add a payment method”.

Thoughts?

**Comment by tofumatt** [comment]
That rule about only three being a category is a good one. I'm going to keep that in my pocket as a good UX line :-)

Credit/Debit is much better, agreed. If there is already a plan to add more payment methods I was thinking the "Payment Methods" text helps us get out in front of it... But mostly I just like how much shorter it is and how much easier to read it felt. Oddly, I think using "or" instead of "/" as the separator allevies a lot of my readability issues with the old text.

So yeah, let's go for "Add Credit/Debit Card or PayPal".
- tofumatt

**Comment by kumar303** [comment]
Also, we don't support PayPal yet :) So we could make it 'Add Credit/Debit Card' for now maybe?

**Comment by brampitoyo** [comment]
Yes. With no PayPal support, the string should be “Add Credit/Debit Card”.
